packages/gnosis-relay-api/src/tests/GnosisRelayAPI.test.ts

```diff
@@ -125,6 +132,12 @@ test('Gnosis Relay API:: Make safe and do transactions', async () => {
     await expect(txHash).not.toEqual(undefined);
 
     console.log(`Waiting on TX: ${txHash}`);
-    await provider.getTransactionReceipt(txHash);
+    const receipt = await provider.getTransactionReceipt(txHash);
+    expect(receipt).toMatchObject({
+      to: safeAddress,
+      status: 1,
+      contractAddress: null,
```

Comment:
I don't know what `contractAddress` is supposed to be so I nailed it down to the value I get. If this should be non-null then there's an issue.

Reply:
That would be if a contract was deployed with the tx so yea should be null here!

Reply:
sweet!